packages/visual-editor/src/components/DraggableBlock/styles.module.css

```diff
@@ -17,9 +16,40 @@
   outline-offset: -2px;
   outline: 2px solid transparent;
   z-index: 1;
+  transition: outline 0.2s;
   pointer-events: none;
 }
 
+.DraggableComponent.Dropzone:before {
+  outline-offset: -1px;
+}
```

Comment:
I think this css change is causing the hover overlay to show as a 1px outline near the edges of the canvas, instead of 2px.

The overlay label appears to be rendering centered in structure components as well:

![Screenshot 2024-06-10 at 9 05 00 AM](https://github.com/contentful/experience-builder/assets/8539634/6d3516f4-f383-40af-b5c6-9023814c05d8)

Comparing with latest development:

![Screenshot 2024-06-10 at 9 04 49 AM](https://github.com/contentful/experience-builder/assets/8539634/fdce32db-159b-4553-a28e-3b9d33037329)

Reply:
Good catch, fixed!